spec/unit/plugins/windows/virtualization_spec.rb

```diff
@@ -18,47 +20,159 @@
 
 require File.expand_path(File.dirname(__FILE__) + '/../../../spec_helper.rb')
 
-describe Ohai::System, "Windows virtualization platform" do
-  let(:plugin) { get_plugin("windows/virtualization")}
+describe Ohai::System, 'Windows virtualization platform' do
+  let(:plugin) { get_plugin('windows/virtualization') }
 
   before(:each) do
     allow(plugin).to receive(:collect_os).and_return(:windows)
-    allow(plugin).to receive(:powershell_exists?).and_return(false)
   end
 
-  describe "when we are checking for parallels" do
-    it "should set parallels guest if powershell exists and it's output contains 'Parallels Software International Inc.'" do
-      allow(plugin).to receive(:powershell_exists?).and_return(true)
-      bios=<<-BIOS
-SMBIOSBIOSVersion : 10.2.0 (28956) rev 0
-Manufacturer      : Parallels Software International Inc.
-Name              : Default System BIOS
-SerialNumber      : Parallels-92 05 B4 56 97 11 4F FA B1 95 1A FF 8E F9 DD CE
-Version           : PRLS   - 1
-      BIOS
-      shellout = double("shellout")
-      allow(shellout).to receive(:stdout).and_return(bios)
-      allow(plugin).to receive(:shell_out).with('powershell.exe "Get-WmiObject -Class Win32_BIOS"').and_return(shellout)
+  context 'when running on vmware fusion' do
```

Comment:
oh looks like I did a select and delete on part of the block before I did the commit :(  Let me push it with the correct test block.